homeassistant/components/xiaomi_miio/remote.py

```diff
@@ -112,22 +117,20 @@ async def async_setup_platform(hass, config, async_add_entities, discovery_info=
 
     async_add_entities([xiaomi_miio_remote])
 
-    async def async_service_handler(service):
+    async def async_service_led_off_handler(entity, service):
+        """Handle set_led_off command."""
+        return device.set_indicator_led(False)
```

Comment:
Nothing is checking this return value.
```suggestion
        device.set_indicator_led(False)
```

Is this method call async safe, ie no I/O?

How do we know that we target the correct device? We need to target the correct device via the selected entity (entity_id).